qiime2/sdk/parallel_config.py

```diff
@@ -138,13 +147,14 @@ def _get_vendored_config():
         elif os.path.exists(fp_ := os.path.join(
                 appdirs.site_config_dir('qiime2'), 'qiime2_config.toml')):
             config_fp = fp_
+        # NOTE: These next two are dependent on us being in a conda environment
         # 4. Check in conda env
         # ~/miniconda3/env/{env_name}/conf
-        elif os.path.exists(fp_ := VENDORED_FP):
+        elif CONDA_PREFIX != '' and os.path.exists(fp_ := VENDORED_FP):
             config_fp = fp_
         # 5. Write the vendored config to the vendored location and use
         # that
-        else:
+        elif CONDA_PREFIX != '':
             with open(VENDORED_FP, 'w') as fh:
```

Comment:
VENDORED_FP will end up being a relative path here `etc/qiime2_config.toml`. Just making sure that's intended.

Reply:
If they are in a conda environment, the conda prefix will be the absolute path to the base of the env  (ex: `/home/anthony/miniconda3/envs/q2-dev`). If we append `etc/qiime2_config.toml` to this we have an absolute path to a specific file. If they are not in a conda environment, there will not be a conda prefix, and we will not try to access this path at all.

Reply:
Misread not equal as equal late on Friday